testsuite/features/step_definitions/command_steps.rb

```diff
@@ -690,9 +690,9 @@
   raise "Command Output #{@command_output} don't include #{arg1}" unless @command_output.include? arg1
 end
 
-When(/^I (start|stop) "([^"]*)" service on "([^"]*)"$/) do |action, service, host|
+When(/^I (start|stop|restart|reload|enable|disable) the "([^"]*)" service on "([^"]*)"$/) do |action, service, host|
```

Comment:
To me the full sentence looks more natural without "the".

Reply:
Example: `I start apache service` vs. `I start the apache service`, where the latter one sounds more grammatically correct to me.

Reply:
Yeah might be me badly translating it from spanish :D Nevermind :) "I start Apache's service" is how I would translated it, but looking other parts of our code, yeah, make sense to add the "the" I guess.